src/panels/ConfigPanel/ConfigPanel.tsx

```diff
@@ -57,12 +57,15 @@ type ConfigQueryParams = {
   modelUUID: string | null;
 };
 
-const generateErrors = (errors: string[], scrollArea?: HTMLElement | null) =>
-  errors.map((error) => (
-    <ScrollOnRender key={error} scrollArea={scrollArea}>
-      <Notification severity="negative">{error}</Notification>
-    </ScrollOnRender>
-  ));
+const generateErrors = (errors: string[], scrollArea: HTMLElement | null) => (
+  <ScrollOnRender scrollArea={scrollArea}>
+    {errors.map((error) => (
+      <Notification key={error} severity="negative">
+        {error}
+      </Notification>
+    ))}
+  </ScrollOnRender>
+);
```

Comment:
Possibly in a follow-up branch this could become a shared component, something like `PanelErrors` and include the `.some(...)` like you do in the other panel, as we're doing the same thing in a number of places.